packages/ai-jsx/src/batteries/use-tools.tsx

```diff
@@ -4,97 +4,9 @@
  * @packageDocumentation
  */
 
-import {
```

Comment:
Removing the prompt-engineered polyfill re @zkoch's suggestion, which I agree with.